### PR TITLE
Add routers for celery tasks.

### PR DESCRIPTION
**Pull request by justinvdm**
Celery allows custom routers for tasks, see https://celery.readthedocs.org/en/latest/userguide/routing.html#routers. I propose adding two custom router classes:
- `CeleryRegexRouter` that directs tasks based on a regular expression for the task name (this is low credit notifications so that they don't need to set behind statement generation and archival tasks)
- `CeleryAppRouter` that directs tasks based on the first two parts of the task name (so all tasks that start with `go.billing.*` are directed to `go.billing`, all tasks that start with `go.conversation.*` go to `go.conversation`).

```
CELERY_ROUTES = (
    CeleryRegexRouter(r'^go\.billing\..*low_credit.*$', queue='go.billing.low_credit'),
    CeleryAppRouter(),
)
```

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review, I think.

File: go/base/celery_routers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    def route_for_task(self, task, args=None, kwargs=None):
+        if self.regex.match(task):
+            return {'queue': self.queue}
+        else:
+            return None
+
+
+class CeleryAppRouter(object):
+    def route_for_task(self, task, args=None, kwargs=None):
+        return {'queue': '.'.join(task.split('.')[:2])}
```
Comment (hodgestar) [inline go/base/celery_routers.py:18]: I wonder if we should return `None` if `task.split('.')` doesn't have enough parts?

Comment (hodgestar) [inline go/base/celery_routers.py:18]: On second thoughts, I think this is fine.

**Comment by smn** [comment]
ooh! :+1:

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**